controllers/GeneralService.php

```diff
@@ -1,8 +1,22 @@
 <?php
+use onboarding\services\RedisService;
 
 class GeneralService
 {
-    function getXMLFromValidation($url, $paramsFromGet)
+    static $cacheName;
```

Comment:
This is more a const than a static field.
In addition, try to think a better name ... what do you think about agentApiKeyRedisKey?